bigtable/hello/main.py

```diff
@@ -68,35 +73,38 @@ def main(project_id, instance_id, table_id):
         #     https://cloud.google.com/bigtable/docs/schema-design
         row_key = 'greeting{}'.format(i)
         row = table.row(row_key)
-        row.set_cell(
-            column_family_id,
-            column_id,
-            value.encode('utf-8'))
-        row.commit()
+        row.set_cell(column_family_id,
+                     column,
+                     value,
+                     timestamp=datetime.datetime.utcnow())
+        rows.append(row)
+    table.mutate_rows(rows)
     # [END writing_rows]
 
     # [START getting_a_row]
-    print('Getting a single greeting by row key.')
+    print 'Getting a single greeting by row key.'
     key = 'greeting0'
-    row = table.read_row(key.encode('utf-8'))
-    value = row.cells[column_family_id][column_id][0].value
-    print('\t{}: {}'.format(key, value.decode('utf-8')))
+
+    # Only retrieve the most recent version of the cell.
+    row_filter = row_filters.CellsColumnLimitFilter(1)
+    row = table.read_row(key, row_filter)
+    print row.cell_value(column_family_id, column)
     # [END getting_a_row]
 
     # [START scanning_all_rows]
-    print('Scanning for all greetings:')
-    partial_rows = table.read_rows()
+    print 'Scanning for all greetings:'
+    partial_rows = table.read_rows(filter_=row_filter)
     partial_rows.consume_all()
```

Comment:
Please do not use `consume_all` as it is deprecated.
`PartialRowsData` is now an iterable. 
Please use
```python
 for row in partial_rows:
```